src/renderer/components/message/MessageList.js

```diff
@@ -41,8 +41,10 @@ export default function MessageList ({ chat, refComposer, locationStreamingEnabl
 
   useEffect(() => {
     if (scrollToBottomIfClose === false) return
-    if (messageListRef.current.scrollHeight - messageListRef.current.scrollTop > 400) {
-      // we are close to bottom so scroll down to show the new message
+    // For whatever reason the scrollTop is ~1000 off...
```

Comment:
_For the protocol: I do not like this "we don't know what its doing lets fix it anyway somehow" approach_